RPI/main.js

```diff
@@ -0,0 +1,14 @@
+const monitor = require("./monitor");
```

Comment:
This is the main file that will be run when RPI started. It'll start the code to monitor sensor reading (L8) and store the reading to db (L9).